lms/djangoapps/teams/views.py

```diff
@@ -169,7 +175,9 @@ class TeamsListView(ExpandableFieldViewMixin, GenericAPIView):
             * topic_id: Filters the result to teams associated with the given
               topic.
 
-            * text_search: Currently not supported.
+            * text_search: Searches for full word matches on the name, description,
```

Comment:
Note in this documentation as well that text_search and order_by can't both be supplied, and that text_search relies on the ENABLE_TEAMS_SEARCH being set to True.